community/jfsutils/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+for patch in *.patch; do
+	patch -p1 < "$patch"
```

Comment:
Something I just noticed; this is a <TAB> not four spaces

Reply:
wasn't aware of conventions, no problem

Reply:
It's fine! The [style guide](https://github.com/kiss-community/wiki/blob/master/kiss/style-guide.txt) was removed from upstream and has yet to be replaced.